### PR TITLE
update pick-right-solution to setup page 

### LINKED ISSUE #14476: Dangling "pick-right-solution" references - Indonesian
This is a Bug Report
Problem:

The link from "Cluster Administration Overview" to " Picking the Right Solution" is a 404.
docs/setup/pick-right-solution/ does not exist, as it was removed as a redundant page.

Proposed Solution:
Update the stale links to point to Setup page not the "Picking the Right Solution" page
The references on this page point both too the root of the `pick-right-solution` page, but also to various IDs that no longer exist, there may be better pages to point those links to.

Page to Update:
https://kubernetes.io/id/docs/setup/
https://kubernetes.io/id/docs/concepts/cluster-administration/cluster-administration-overview/

### PR DESCRIPTION
**Pull request by medmedchiheb**
Fixes #14476

### CODE REVIEW

File: content/id/docs/concepts/cluster-administration/cluster-administration-overview.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Diharapkan untuk memahami beberapa [konsep](/docs/concepts/) dasar Kubernetes se
 {{% capture body %}}
 ## Perencanaan Kluster
 
-Lihat panduan di [Memilih Solusi yang Tepat](/docs/setup/pick-right-solution/) untuk mempelajari beberapa contoh tentang bagaimana merencanakan, mengatur dan mengonfigurasi kluster Kubernetes. Solusi yang akan dipaparkan di bawah ini disebut *distro*.
+Lihat panduan di [Mempersiapkan](/docs/setup) untuk mempelajari beberapa contoh tentang bagaimana merencanakan, mengatur dan mengonfigurasi kluster Kubernetes. Solusi yang akan dipaparkan di bawah ini disebut *distro*.
```
Comment (girikuncoro) [inline content/id/docs/concepts/cluster-administration/cluster-administration-overview.md]: "Mempersiapkan" should be "Persiapan" to be consistent with the /docs/setup headings

**Comment by girikuncoro** [comment]
Thanks for fixing this, can you help addressing my comment?

File: content/id/docs/concepts/cluster-administration/cluster-administration-overview.md (hunk 1 above)
Comment (medmedchiheb) [reply content/id/docs/concepts/cluster-administration/cluster-administration-overview.md]: Alright

**Comment by girikuncoro** [comment]
@medmedchiheb thanks for your PR!

/lgtm
/approve

**Pull request merged**